core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-reports/dot-experiments-reports.component.html

```diff
@@ -17,142 +17,25 @@
-                                    class="flex col-fixed flex-wrap align-content-center justify-content-end"
-                                    [ngClass]="{ 'prospect-winner': row.isWinner }"
-                                    style="width: 150px"
-                                >
-                                    {{ row.conversionRate }}
-                                </div>
-                                <div
-                                    class="flex col flex-wrap align-content-center justify-content-end"
-                                    [ngClass]="{ 'prospect-winner': row.isWinner }"
-                                >
-                                    {{ row.probabilityToBeBest }}
-                                </div>
-                                <div
-                                    class="flex col flex-wrap align-content-center justify-content-end"
-                                    [ngClass]="{ 'prospect-winner': row.isWinner }"
-                                >
-                                    {{ row.conversionRateRange }}
-                                </div>
-                                <div
-                                    class="flex col flex-wrap align-content-center justify-content-end"
-                                    [ngClass]="{ 'prospect-winner': row.isWinner }"
-                                >
-                                    <button
-                                        class="p-button-sm p-button-secondary"
-                                        *ngIf="!vm.promotedVariant && row.id !== defaultVariantId"
-                                        [label]="'experiments.reports.promote' | dm"
-                                        (click)="promoteVariant($event, vm.experiment.id, row)"
-                                        data-testId="promote-variant-button"
-                                        pButton
-                                        type="button"
-                                        icon="pi pi-send"
-                                    ></button>
-                                    <p-tag
-                                        class="sm p-tag-info"
-                                        *ngIf="row.isPromoted"
-                                        [value]="'promoted' | dm | titlecase"
-                                        data-testId="variant-promoted-tag"
-                                        icon="pi pi-send"
-                                    />
-                                </div>
-                            </div>
-                        </ng-template>
-                    </dot-experiments-details-table>
-                </div>
+                            <dot-experiments-report-daily-details
+                                [detailData]="vm.detailData"
+                                [experimentId]="vm.experiment.id"
+                                [hasEnoughSessions]="!vm.hasEnoughSessions"
```

Comment:
Out of curiosity, why does the input for `hasEnoughSessions` depend on `vm.hasEnoughSessions` being falsy?